test/built-ins/Number/prototype/toFixed/return-values.js

```diff
@@ -0,0 +1,12 @@
+// Copyright (C) 2016 The V8 Project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+es6id: 20.1.3.3
+esid: sec-number.prototype.tofixed
+description: >
+  Return regular string values
+---*/
+
+// a bug in IE ? - Edge 14
```

Comment:
this is vague, but a link reference wouldn't help it as well. The bug case seems as a good reason for this addition but I would add it as a commit description, not to the code.

Do you mind to move this to the commit description providing more info if any?